template/scripts/drupal-vm/config.yml

```diff
@@ -0,0 +1,30 @@
+# Update the hostname to the local development environment hostname
+vagrant_hostname: ${project.machine_name}.localhost
```

Comment:
It really should be `${project.local_uri}`—see https://github.com/acquia/blt/pull/113/files#diff-1d79176dc01e3cdc73b4066200be0e02R2